src/stores/analyticsContext.test.tsx

```diff
@@ -416,7 +416,7 @@ describe('Analytics context', () => {
         'setDocumentTitle',
         'Test Doc Title',
       ])
-      expect(windowWithAnalytics._paq[1]).toEqual(['setDoNotTrack', true])
+      expect(windowWithAnalytics._paq[1]).toEqual(['setDoNotTrack', false])
```

Comment:
We talked about this during one of our meetings and decided it should be a launch darkly flag. Suggestion is to name it `MatomoClientSideDoNotTrack` and have it return true or false to be set here.

Reply:
After initial investigation this wasn't working, not sure that LD is initialized early enough for it two work. So we decided to just leave it as is for now so we can confirm it allows us to track rank and location properly.